src/en/guide/basics/headers.md

```diff
@@ -25,23 +31,57 @@ $ curl localhost:8000 \
     -H "Authorization: Token ABCDEF12345679"
 ABCDEF12345679
 ```
+
 ```bash
 $ curl localhost:8000 \
     -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
 ```
+
 :---
+
 #### Proxy headers
+
 Sanic has special handling for proxy headers. See the [proxy headers](/guide/advanced/proxy-headers.md) section for more details.
 
 ---:1
+
+#### Other headers
+
+Sanic support you operate headers, Header keys are converted to *lowercase* when parsed. Capitalization is not considered for headers.
+You can operate it like dict.
+
+:--:1
+
+```python
+@app.route("/")
+async def handler(request):
+    print(request.headers["user-agent"])
+    print(request.headers.items())
+    return json(dict(request.headers))
```

Comment:
Let's use this example as it more clearly illustrates the variety of methods, and how the object can be accessed.

```python
    return json(
        {
            "foo_weakref": request.headers["foo"],
            "foo_get": request.headers.get("Foo"),
            "foo_getone": request.headers.getone("FOO"),
            "foo_getall": request.headers.getall("fOo"),
            "all": list(request.headers.items()),
        }
    )
```